src/GitLink.Tests/GlobalInitialization.cs

```diff
@@ -1,17 +1,16 @@
 ﻿// --------------------------------------------------------------------------------------------------------------------
 // <copyright file="GlobalInitialization.cs" company="CatenaLogic">
-//   Copyright (c) 2014 - 2014 CatenaLogic. All rights reserved.
+//   Copyright (c) 2014 - 2016 CatenaLogic. All rights reserved.
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
-
 using Catel.Logging;
 using NUnit.Framework;
 
 [SetUpFixture]
 public class GlobalInitialization
 {
-    //[SetUp]
+    ////[SetUp]
```

Comment:
Can we just disable / ignore the rule? I mean, I prefer spaces for single line comments as well, but rules shouldn't get in the way and we should be able comment some code.